_config.yml

```diff
@@ -138,6 +144,9 @@ collections:
     output: true
     permalink: /:collection/:name/
 
+include: 
+  - _stories
```

Comment:
This one is interesting. Quarto does NOT allow you to really change where it wants to put it's supporting files, it collocates them next to the source file... which is fine but Jekyll ignores any `_` directories by default, so we poke a hole to push all the Quarto figures and diagrams through.